q1k_sync_tools.py

```diff
@@ -2,46 +2,52 @@
-    event_dict = {}
-    for i in range(0,len(stim_names)-1):
-        event_dict[stim_names[i]] = i + 1
-
-    ##method for building building the event_dict is precarious, but it seems to satisfy all of the cases... to be reworked later
-    #if raw.info.ch_names[-1] == 'VBeg':
-    #    print('VBeg method')
-    #    adjuster = len(raw.info.ch_names) - 129 - events[0,2] # this assumes that 'VBeg' is the last stim channel and the first event in the recording.
-    #    event_dict = {}
-    #    for i in range(129,len(raw.info.ch_names)):
-    #        event_dict[raw.info.ch_names[i]] = i-129 + 1 - adjuster
-
-    #if raw.info.ch_names[-1] == 'STI 014':
-    #    print('STI 014 method')
-    #    #adjuster = len(raw.info.ch_names) - 130 - events[1,2] # this assumes that 'STI 014' is the last stim channel and the first event in the recording.
-    #    event_dict = {}
-    #   for i in range(129,len(raw.info.ch_names)):
-    #        event_dict[raw.info.ch_names[i]] = i-129 + 1# - adjuster
-
-    ## check for dstr label and if found remove it
-    #try:
-    #    event_dict['dstr']
-    #    print('found dstr label.. removing it')
-    #    del event_dict['dstr']
-    #    for k in event_dict:
-    #        event_dict[k] -= 1
-    #except:
-    #    print('no dstr label found.. continuing')
-
-    return event_dict               
-
-
-def eeg_event_test(eeg_events, eeg_event_dict, task_name=''):
+VALID_TASKS = ['rest', 'as', 'ssvep', 'vs', 'ssaep',
+               'go', 'plr', 'mmn', 'nsp', 'fsp']
+
+
+def get_event_dict(raw, events):
+
+    stim_names = raw.copy().pick('stim').info['ch_names']
+    event_dict = {event: int(i) + 1
+                  for i, event in enumerate(stim_names)
+                  if event != 'STI 014'}
+
+    """ #method for building building the event_dict is precarious, but it seems to satisfy all of the cases... to be reworked later
+    if raw.info.ch_names[-1] == 'VBeg':
+        print('VBeg method')
+        adjuster = len(raw.info.ch_names) - 129 - events[0,2] # this assumes that 'VBeg' is the last stim channel and the first event in the recording.
+        event_dict = {}
+        for i in range(129,len(raw.info.ch_names)):
+            event_dict[raw.info.ch_names[i]] = i-129 + 1 - adjuster
+
+    if raw.info.ch_names[-1] == 'STI 014':
+        print('STI 014 method')
+        #adjuster = len(raw.info.ch_names) - 130 - events[1,2] # this assumes that 'STI 014' is the last stim channel and the first event in the recording.
+        event_dict = {}
+       for i in range(129,len(raw.info.ch_names)):
+            event_dict[raw.info.ch_names[i]] = i-129 + 1# - adjuster
+
+    # check for dstr label and if found remove it
+    try:
+        event_dict['dstr']
+        print('found dstr label.. removing it')
+        del event_dict['dstr']
+        for k in event_dict:
+            event_dict[k] -= 1
+    except:
+        print('no dstr label found.. continuing')"""
+
+    return event_dict            
+
```

Comment:
Just wrapping that whole commented section in triple quotes instead of having a `#` on each line.